src/components/dashboard/SafeAppsDashboardSection/SafeAppsDashboardSection.tsx

```diff
@@ -41,13 +42,15 @@ const ExploreSafeAppsCard = () => {
   const safeAppsLink = `${AppRoutes.apps}?safe=${router.query.safe}`
 
   return (
-    <AppCardContainer url={safeAppsLink}>
```

Comment:
Won't this make the card not clickable?

Reply:
Yes, now only the button itself is clickable. Previously, both the button and the card were clickable which could be confusing especially when using the keyboard to navigate.

Reply:
I thought it was a feature and a good one.

Reply:
I can revert it but would suggest we look into a better solution e.g. converting the button into a title that is not focusable/clickable to avoid having the same user action twice.